Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +627,16 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private updateInactiveRegions(params: InactiveRegionParams): void {
+        let renderOptions: vscode.DecorationRenderOptions = {
+            color: "rgba(255,0,255,0.5)"
+        };
+        let textEditorDecoration = vscode.window.createTextEditorDecorationType(renderOptions);
```

Comment:
>let textEditorDecoration = vscode.window.createTextEditorDecorationType(renderOptions); [](start = 8, length = 87)

We probably only need to create this object once when the class is first initialized.